ingestion/functions/parsing/colombia/colombia.py

```diff
@@ -17,12 +17,13 @@
     import parsing_lib
 
 
-# Load manual geocoding dictionaries, made from Colombia municipality -> coordinate lookup table
```

Comment:
Again I think some comments here might be helpful regarding what these dicts are and where they are from.